lib/IRGen/IRGenSIL.cpp

```diff
@@ -5103,15 +5103,18 @@ void IRGenSILFunction::emitDebugInfoForAllocStack(AllocStackInst *i,
   assert(isa<llvm::AllocaInst>(addr) || isa<llvm::UndefValue>(addr) ||
          isa<llvm::IntrinsicInst>(addr) || isCallToSwiftTaskAlloc(addr));
 
-  auto Indirection = DirectValue;
+  bool InCoroContext =
```

Comment:
Consider factoring this into a hasCoroContext(SILFunction &, SILInstruction &) helper?

Reply:
Good point!

Reply:
Done.